docs/docs/guides/authentication/jwt.md

```diff
@@ -0,0 +1,136 @@
+# `JWT`
+
+## Example Configuration
+
+## JWT Reference
+
+| Name            | Description                                                         | Type   | 
+| :-------------- | ------------------------------------------------------------------- |----
+| `auth.jwt.providers`      |Providers to use for verifying JSON Web Tokens (JWTs) on the virtual host.                                                                     | list
+
+
+Provider properties:
+| Name            | Description                                                         | Type   | Required 
+| :-------------- | ------------------------------------------------------------------- |----|-
+| `provider.name`      |Unique name for the provider.                                                           | string | true
+| `provider.issuer`      | Issuer that JWTs are required to have in the "iss" field. If not provided, JWT issuers are not checked.                                                                    | string | true
+| `provider.jwks` | Remote JWKS to use for verifying JWT signatures. The URI for the JWKS. | URI | true
+| `provider.default`      |      Whether the provider should apply to all	routes in the HTTPProxy/its includes by	default. At most one provider can be marked 	as the default. If no provider is marked as the default, individual routes must explicitly	identify the provider they require.                                                               | boolean | false
+| `provider.audiences`| Audiences that JWTs are allowed to have in the "aud" field.If not provided, JWT audiences are not checked. | list | false
+| `provider.forward` | Whether the JWT should be forwarded to the backend service after successful verification. By default, the JWT is not forwarded. | boolean | false
+
```

Comment:
Markdown table looks like this when rendered in HTML

![image](https://user-images.githubusercontent.com/8327751/203776424-88024312-89d0-48c4-9fbd-b35430c1eb9b.png)

@aabedraba I don't know if this format will fit what we have but it should be plenty for expanding docs. Your call if you want to use it or not